app/assets/stylesheets/common.scss

```diff
@@ -764,6 +764,10 @@ header .search_forms,
     overflow: hidden;
     border-radius: 2px 0 0 2px;
   }
+  
+  input#query {
```

Comment:
Now that you've moved I see we already have rules for the input field with the `input[type=text]` selector - this rule should probably just be in there unless there is a good reason not to? Does that other rule select other things?

Reply:
I don't see any reason not to. Currently we only have this one field with the `overflow` class and even if there was another field it would make sense to let it also behave like that, so I moved the CSS rule there.